src/map/handler/Map.TouchZoom.js

```diff
@@ -121,6 +125,24 @@ export var TouchZoom = Handler.extend({
 		} else {
 			this._map._resetView(this._center, this._map._limitZoom(this._zoom));
 		}
+
+		this._enableMapDrag();
+	},
+
+	_disableMapDrag: function () {
```

Comment:
"Disable" Map Drag with starting pinch zoom and enable the drag after pinch zoom makes it possible to drag the map immediately after zooming.